### PR TITLE
Fun with cli cm

### PR DESCRIPTION
**Pull request by morecallan**
@jcockhren @katerebekah - Can't help myself.

### CODE REVIEW

**Comment by guthb** [comment]
@morecallan should we check on the first entry that we are requesting one of the 5 selections when we prompt the user?  It accepts any value, I entered "6" and then steps into the print request ( which does check for integers).  I like that we now kill the program by typing "Shut up" in the how many field.

**Comment by morecallan** [comment]
@guthb If the product owners are cool with us making an improvement to their desired flow, this would be much better UX. 

PS. If you enter "shut up" in either field, the math wizard bounces. 🎱

**Comment by guthb** [comment]
@morecallan I checked again,  i missed that it needed to be all lowercase, I thought it needed to be 'S' character in "shut up".  I would like to wrap it in the string .ToLower method.

**Comment by morecallan** [comment]
@guthb WORD. did it.

**Comment by guthb** [comment]
@morecallan  I corrected the branch issue by  executing git checkout -b FunWithCLI-cm origin/
FunWithCLI-cm.  I see that the case concern is corrected.

**Comment by guthb** [comment]
👍

File: Mathmagician/Extras/ASCII.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician.Extras
+{
+    public static class ASCII
+    {
+        public static string Wizard()
+        {
+            return "              o\r\n"
```
Comment (katerebekah) [inline Mathmagician/Extras/ASCII.cs:13]: :+1:

File: Mathmagician/Program.cs [hunk 2]
```diff
@@ -3,33 +3,80 @@
...
+                    listAmountInput = Console.ReadLine();
 
+                    //Checks to see if user entered valid number, if they instruct to quit, then exit loop and close program, if not, then throw error and repeat instruction.
+                    if (Int32.TryParse(listAmountInput, out listAmountParsed))
+                    {
+                        responseUnverified = false;
+                    }
+                    else
+                    {
+                        if (listAmountInput.ToLower() == "shut up")
```
Comment (katerebekah) [inline Mathmagician/Program.cs:62]: You make this check twice. In a future refactor, you might want to extract this checking functionality into method.

**Comment by katerebekah** [comment]
:+1:

**Pull request merged**